packages/angular_devkit/build_angular/BUILD.bazel

```diff
@@ -236,15 +244,22 @@ pkg_npm(
     ],
 )
 
-api_golden_test_npm_package(
-    name = "build_angular_api",
-    data = [
-        ":npm_package",
-        "//goldens:public-api",
-    ],
-    golden_dir = "angular_cli/goldens/public-api/angular_devkit/build_angular",
-    npm_package = "angular_cli/packages/angular_devkit/build_angular/npm_package",
-)
+[
+    api_golden_test_npm_package(
```

Comment:
I don't think we should run any of the api_golden tests on multi platform. It's not like the API golden file will change based on OS.

Reply:
Ok these ones have been reverted.

a9c9bbd1507c30cdd3b6a01d36f2f7fbfbf60191 for reference to the original that does change these